### ISSUE TITLE
Fix scaling on Traitor mission failure penalty

### ISSUE THREAD

**Issue by aefreedman** [nateberkopec/antistasi#69]
In `as_traidor.sqf` the penalty for ignoring the mission is losing 1/3 of your HR and money.

```
  if (dateToNumber date > _fechalimnum) then
    {
    _hrT = server getVariable "hr";
    _resourcesFIAT = server getVariable "resourcesFIA";
    [-1*(round(_hrT/3)),-1*(round(_resourcesFIAT/3))] remoteExec ["resourcesFIA",2];
    }

**Comment by aefreedman** [comment]
Traitor is the only instance of a scaled penalty. This should never happen in this game _unless_ the penalty for failure is very clearly indicated.

**Comment by nateberkopec** [comment]
Seems to me the Defend Petros mission is bad enough. If Defend Petros cannot be started (due to aaf counterattack active or whatever the condition is), then there should be an appropriately bad penalty I think.

**Comment by aefreedman** [comment]
Yeah my thinking, too. In general rewards and penalties should be 1:1 -- you complete a mission you get a single coherent reward. Same for failure.
